api/views/token_use.py

```diff
@@ -1,37 +1,25 @@
-import re
-
 from django.shortcuts import get_object_or_404
```

Comment:
you imported the wrong get_object_or_404, please use the rest_framework version for APIs.